### DISCUSSION TITLE
How do I add a line on the seekbar?

### DISCUSSION THREAD

**Discussion by backduu** [videojs/video.js#8424] [Q&A]
### Description

First of all, I just tried to represent watched buffer. like below picture.

![image](https://github.com/videojs/video.js/assets/59520329/d7baf1b7-7a4b-4316-865e-118229dbffd0)

![image](https://github.com/videojs/video.js/assets/59520329/d16a1b6c-0913-4a3b-88ad-38d1592e602b)

### Reduced test case

https://sorry..

### Steps to reproduce

1. add a tags for addition line on the seekbar
2. there's some errors due to the css
3. looking for a code or a simple example to make custom component on the seekbar or controlbar

### Errors

but there are some errors due to the css.

so i was googling.

there is a way to add a custome component like a bar on the ControlBar.

however....i am a beginner. so can you give me an advise for adding component??

### What version of Video.js are you using?

7.11.5

### What browser(s) including version(s) does this occur with?

Chrome 116

### What OS(es) and version(s) does this occur with?

Window 11

**Comment by welcome** [comment]
👋 Thanks for opening your first issue here! 👋

If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
To help make it easier for us to investigate your issue, please follow the [contributing guidelines](https://github.com/videojs/video.js/blob/master/CONTRIBUTING.md).

**Comment by amtins** [comment]
@backduu I've created a Gist showing how this can be achieved, see https://gist.github.com/amtins/49ce4e97ef46e24eb1ef23b24a4e83e4

The function that helps to do this is `player.played`
